visualization/app/codeCharta/ui/matchingFilesCounter/matchingFilesCounter.component.ts

```diff
@@ -29,26 +27,25 @@ export class MatchingFilesCounterController implements NodeSearchSubscriber, Bla
 
 	public onNodeSearchComplete(searchedNodes: CodeMapNode[]) {
 		this.searchedNodeLeaves = this.getSearchedNodeLeaves(searchedNodes)
-		this.updateViewModel(this.searchedNodeLeaves, this._viewModel.blacklist)
+		this.updateViewModel(this.searchedNodeLeaves)
 	}
 
 	public onBlacklistChanged(blacklist: BlacklistItem[]) {
-		this._viewModel.blacklist = blacklist
-		this.updateViewModel(this.searchedNodeLeaves, this._viewModel.blacklist)
+		this.updateViewModel(this.searchedNodeLeaves)
 	}
```

Comment:
onBlacklistChanged the fileCounters are not updated.
- Search for `*Parent*`
- Open the TreeView
- Flatten the node `smallLeaf.html`
- => now the flattenedFileCounter is not updated to 1

Reply:
fixed